actionpack/lib/action_controller/base.rb

```diff
@@ -87,10 +87,10 @@ module ActionController
   #
   # or you can remove the entire session with +reset_session+.
   #
-  # Sessions are stored by default in a browser cookie that's cryptographically signed, but unencrypted.
-  # This prevents the user from tampering with the session but also allows them to see its contents.
+  # Sessions are stored by default(ActionDispatch::Session::CookieStore) in a browser cookie that's encrypted.
+  # The client cannot read or edit the contents of the cookie, without breaking encryption.
```

Comment:
I realize this is the [same wording as in the Security Guide](https://guides.rubyonrails.org/v7.0.0/security.html#session-storage), but what do you think about tweaking it a bit?

```suggestion
  # By default, sessions are stored in an encrypted browser cookie (see
  # ActionDispatch::Session::CookieStore). Thus the user will not be able to
  # read or edit the session data. However, the user can keep a copy of the
  # cookie even after it has expired, so you should avoid storing sensitive
  # information in cookie-based sessions.
```